_data/projects/Webber.yml

```diff
@@ -0,0 +1,10 @@
+name: Webber
+desc: a WYSIWYG editor for html
+site: https://github.com/dragon540/Webber
+tags:
+- qt
+- cplusplus
+- wysiwyg
+upforgrabs:
+  name: help needed
+  link: https://github.com/dragon540/Webber/issues
```

Comment:
I think this should be pointed to your `help wanted` label: `https://github.com/dragon540/Webber/labels/help%20wanted`

Reply:
@dragon540 You should change the link as mentioned by @shiftkey

Reply:
@dragon540 please make this change so that I could merge.